en/pe-docs/docs/administer/configure-self-sign-up.md

```diff
@@ -0,0 +1,136 @@
+# Configure Self-Sign-Up
```

Comment:
As per stefenie, we do not have this covered in PE view. Let's keep an open issue related to docs and remove this from here.